### PR TITLE
Add Open Graph meta tags

### PR DESCRIPTION
**Pull request by DatGuy1**
Tested with https://www.opengraph.xyz/url/

Also, I feel like there should be a HTML formatter, but I suppose it would get very confused with the Flask stuff

Pull request checklist:

- [X] `CHANGELOG.md` was updated, if applicable
- [X] I have tested all changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 
 - Minor: Add a global command cooldown module. This allows you to share a cooldown between selected commands. (#1714)
 - Minor: Add the QueUp module back. (#1570)
+- Minor: Added Open Graph metadata to improve look in embeds such as in Twitter and Discord. (#1721)
```
Comment (ALazyMeme) [inline CHANGELOG.md:9]: According to the opengraph website you linked, Twitter uses it's own meta tags (not opengraph ones). Does it actually work for Twitter?

File: templates/layout.html [hunk 2]
```diff
@@ -4,6 +4,10 @@
     <meta name=viewport content="width=device-width, initial-scale=1">
 <title>{{ streamer.name }} - {% block title %}Title{% endblock %}</title>
 <meta charset="utf-8">
+<meta property="og:title" content="{{ streamer.name }} - {{ self.title() }}" />
+<meta property="og:type" content="website" />
+<meta property="og:image" content="/static/images/logo_{{ streamer.full_name }}.png" />
+<meta property="og:description" content="{{ self.title() }} page in {{ bot.name }}'s website" />
```
Comment (ALazyMeme) [inline templates/layout.html:10]:
```suggestion
<meta property="og:description" content="{{ self.title() }} page for {{ bot.name }}'s website" />
```

**Comment by ALazyMeme** [comment]
The image on twitter seems to be the only thing that doesn't work for me
image

**Comment by DatGuy1** [comment]
I'm unsure. I know Twitter does have their own meta tags using a similar specification (for example, [in this very PR](https://i.imgur.com/umQVqX0.png)), but according to their [docs](https://developer.twitter.com/en/docs/twitter-for-websites/cards/guides/getting-started) it should fallback to Open Graph.

**Review by pajlada — APPROVED** [review]

**Pull request merged**